babel.config.js

```diff
@@ -10,19 +10,19 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        targets: '> 0.5%, last 2 versions, Firefox ESR, not dead'
+        // https://browsersl.ist/#q=%3E0.3%25%2C+last+4+versions%2C+not+dead%2C+Firefox+ESR%2C+not+and_qq+13-14%2C+not+kaios+2-5%0A
+        targets: '>0.3%, last 4 versions, not dead, Firefox ESR, not and_qq 13-14, not kaios 2-5'
       }
     ],
-    '@babel/preset-react',
+    [
+      '@babel/preset-react',
+      {
+        useBuiltIns: true // https://babeljs.io/docs/babel-preset-react#usebuiltins
+      }
+    ], // https://babeljs.io/docs/babel-preset-react#usebuiltins
     ['@babel/preset-typescript', { onlyRemoveTypeImports: true }]
   ],
-  plugins: [
-    '@babel/plugin-transform-object-assign',
```

Comment:
Natively supported by targeted browsers: https://caniuse.com/mdn-javascript_builtins_object_assign

Reply:
I love that you're going through these. the babel config has needed serious love. lol

Reply:
Big plus one. Remember to remove deleted plugins from `package.json`.